mappings/net/minecraft/server/command/TestCommand.mapping

```diff
@@ -86,7 +86,7 @@ CLASS net/minecraft/class_4527 net/minecraft/server/command/TestCommand
 		ARG 0 context
 	METHOD method_22295 (Lcom/mojang/brigadier/context/CommandContext;)I
 		ARG 0 context
-	METHOD method_23647 setWorld (Lnet/minecraft/class_4529;Lnet/minecraft/class_3218;)V
+	METHOD method_23647 executeBeforeBatchConsumer (Lnet/minecraft/class_4529;Lnet/minecraft/class_3218;)V
```

Comment:
Maybe just `beforeBatch`?
```suggestion
	METHOD method_23647 beforeBatch (Lnet/minecraft/class_4529;Lnet/minecraft/class_3218;)V
```
Lol github why do you eat my comment

Reply:
I used a name with `execute` since it matches the other names in the class, but that works as well.